src/main/java/com/mathworks/ci/MatlabExecutionException.java

```diff
@@ -1,5 +1,10 @@
 package com.mathworks.ci;
 
+/**
+ * Copyright 2021 The MathWorks, Inc.
```

Comment:
I had a look at previous releases and found it existed since 2021 ([commit history](https://github.com/mathworks/jenkins-matlab-plugin/commits/matlab-2.13.0/src/main/java/com/mathworks/ci/MatlabExecutionException.java) of file before refactoring)